source/content/guides/edge-integrations/03-module-install.md

```diff
@@ -1,29 +1,134 @@
 ---
 title: Edge Integrations
-subtitle: Install Drupal Module
-description: Install, configure, and use the Edge Integrations Drupal module.
+subtitle: Drupal Installation
+description: Install, configure, and use Edge Integrations with Drupal.
 categories: [develop]
 tags: [collaborate, composer, continuous-integrations, webops, workflow]
-contributors: [michellecolon-pantheon, jazzsequence, jspellman814]
+contributors: [michellecolon-pantheon, jazzsequence, jspellman814, robloach, enotick]
 type: guide
 anchorid: module-install
 permalink: docs/guides/edge-integrations/module-install/
 editpath: edge-integrations/03-module-install.md
-reviewed: "2021-11-10"
+reviewed: "2021-03-01"
 ---
 
-Overview-type stuff here.
+This guide is made to facilitate the onboarding process for developers who are implementing content personalization via Pantheon's Advanced Global CDN into their own Drupal website. The personalization system consists of two main parts:
 
-## Where to get the thing and how to install the module
+- Pantheon Edge Integrations library - a CMS agnostic PHP library that allows reading, process, and set VARY headers to be recognized by Pantheon Advanced Global CDN
+- A collection of Drupal 8+ modules in the [Smart Content](https://www.drupal.org/project/smart_content) ecosystem that provide Smart Personalization blocks and preview functionality
 
 
+## Initial setup
 
-## How to configure Geolocation
+1. Ensure you have a working Drupal 8+ installation ready
+1. Install [Smart Content CDN module](https://github.com/pantheon-systems/smart_content_cdn) and its dependencies, which can be pulled with Composer from their corresponding Pantheon repositories. This includes:
+    - [Pantheon Edge Integrations PHP library](https://github.com/pantheon-systems/pantheon-edge-integrations)
+    - [Smart Content](https://www.drupal.org/project/smart_content)
+    ```
+    composer require pantheon-systems/smart_content_cdn
+    ```
+1. Enable the Smart Content CDN module
```

Comment:
Hi @RobLoach - Are there screenshots of any of this- and the the following- steps anywhere? Happy to take the screenshots if pointed in the right direction as well 🙂